src/game_interpreter.cpp

```diff
@@ -2504,6 +2504,9 @@ bool Game_Interpreter::CommandShowPicture(RPG::EventCommand const& com) { // cod
 		// transparency. >= 1.12 Editor only let you set one transparency field but it affects
 		// both chunks here.
 		params.bottom_trans = com.parameters[14];
+	} else if (Player::IsRPG2k3() && !Player::IsRPG2k3E()) {
+		// Corner case when 2k maps are used in 2k3 (pre-1.10) and don't contain this chunk
+		params.bottom_trans = params.top_trans;
```

Comment:
Don't think it matters realistically, but it might be cleaner to do it as:

```
params.bottom_trans = params.top_trans;
if (param_size > 14) {
    params.bottom_trans = com.parameters[14];
}
```

Then at least we're guaranteed `bottom_trans` is always initialized to a sane value.

Reply:
The old code that got lost by your commit was ``params.bottom_trans = param_size > 14 ? com.parameters[14] : params.top_trans;``. So is exactly this one. I can add it back.

Not sure how much impact it has on "LSD compatible fixes for SavePicture::current_bot_trans". I prefer working interpreter over RPG_RT compatible LSDs.

Reply:
Ahh no you're right. If we do that then this chunk will start showing up in 2k saves. Your version is correct.